tools/ci_build/github/azure-pipelines/orttraining-linux-ci-pipeline.yml

```diff
@@ -13,7 +13,6 @@ jobs:
       --build_wheel
       --config $(buildConfig) 
       --enable_training
-      --use_dnnl
```

Comment:
Given it's temporary would it be better to just comment this out instead of deleting it?

Reply:
Sounds good. This time I'll try to get this change in first, and I'll track the change and be responsible to get it back before the next release. The next time I'll follow your suggestion.